server/utils/formUtils.ts

```diff
@@ -173,17 +174,31 @@ export function convertKeyValuePairsToSummaryListItems<T extends object>(
   return Object.keys(values).map(key => summaryListItem(titles[key], String(values[key as keyof T])))
 }
 
+type RenderAs = keyof TextItem | keyof HtmlItem | 'textBlock' | 'date'
+
+const renderSummaryValue = (value: string, renderAs: RenderAs) => {
+  switch (renderAs) {
+    case 'textBlock':
+      return { html: `<span class="govuk-summary-list__textblock">${value}</span>` }
+    case 'date':
+      return { text: (value && DateFormats.isoDateToUIDate(value)) || '' }
+    case 'html':
+      return { html: value }
+    default:
+      return { text: value }
+  }
+}
+
 export const summaryListItem = (
   label: string,
   value: string,
-  renderAs: keyof TextItem | keyof HtmlItem | 'textBlock' = 'text',
-  supressBlank = false,
+  renderAs: RenderAs = 'text',
+  suppressBlank = false,
```

Comment:
Maybe replace these with a single `options` object, to avoid having to pass unused `undefined` values for `renderAs`? Although it would make most uses more verbose...

Reply:
Yes, I thought about that and couldn't decide if it was the right way to go. 
I think the vast majority of cases don't want to suppress blanks, so maybe a wrapper function would be better.